README.md

```diff
@@ -1480,6 +1480,10 @@ For each **table** in `tables` property:
 14. `deleteFields` / `addFields` - delete (or add with a default value), a field at the specified path for all JSON entries in the table
 15. `populateJsonWithId` - This schema.json entry and the disable option is no longer supported. The primary key is always copied into `jsonb->'id'` on each insert and update.
 16. `pkColumnName` - No longer supported. The name of the primary key column is always `id` and is copied into `jsonb->'id'` in each insert and update. The method PostgresClient.setIdField(String) no longer exists.
+17. `withOptimisticLocking` - Creates insert/update trigger to auto populate/update `_version` field in the json
+    * `off` Optimistic Locking is disabled. The trigger does nothing
+    * `logOnConflict` Optimistic Locking is enabled. Version conflict info is logged as a warning message
```

Comment:
logOnConflict documentation needs some explanation how RMB based modules need to configure slf4j/log4j2 to get this logged. You can handle this within this PR or create a follow-up Jira for this.

Reply:
The SQL error code 23F09 will be logged, this should be mentioned here to help filtering the logs.

Reply:
> logOnConflict documentation needs some explanation how RMB based modules need to configure slf4j/log4j2 to get this logged. You can handle this within this PR or create a follow-up Jira for this.

Not sure what config needs to be tweaked, but I was able to see the WARN message. I added a sample to the doc.